x-pack/docs/en/security/securing-communications/tls-versions-jdk.asciidoc

```diff
@@ -0,0 +1,119 @@
+[role="xpack"]
+[[jdk-tls-versions]]
+=== Supported SSL/TLS versions by JDK version
+
+{es} relies on your JDK's implementation of SSL and TLS.
+
+Different JDK versions support different versions of SSL, and this may affect how {es} operates.
+
+`SSLv3`::
+  SSL v3 is supported on all JDKs, but is disabled by default.
+  See <<jdk-enable-tls-protocol>>
+
+`TLSv1`::
+  TLS v1.0 is supported on all JDKs.
+  Some newer JDKs, including the JDK bundled with {es} disable TLS v1.0 by default
+  See <<jdk-enable-tls-protocol>>
+
+`TLSv1.1`::
+  TLS v1.1 is supported on all JDKs.
+  Some newer JDKs, including the JDK bundled with {es} disable TLS v1.1 by default
+  See <<jdk-enable-tls-protocol>>
+
+`TLSv1.2`::
+  TLS v1.2 is supported on all JDKs.
+  It is enabled by default on all JDKs that are supported by {es}, including the bundled JDK.
+
+`TLSv1.3`::
+  TLS v1.3 is supported on all recent JDKs. This includes the latest release of Java 8
+  (any release newer than 8u261) as well as all versions of Java 11 or later.
+  It is supported and enabled by default on the {es} bundled JDK.
+
+NOTE: The information above applies when running on the default JSSE provider in the JDK.
+      JVMs that are configured to use a <<fips-140-compliance, FIPS 140-2>> security
+      provider may have a custom TLS implementation and the supported TLS protocol versions
+      may differ from the list above.
+      +
+      Please check the release notes for your security provider.
+
+[[jdk-enable-tls-protocol]]
+==== Enabling additional SSL/TLS versions on your JDK
+
+The set of supported SSL/TLS versions for a JDK is controlled by a configuration file.
+It is possible to override values that are set in this configuration file in order to
+enable legacy SSL or TLS versions.
```

Comment:
I'm worried that the tongue twister is hard to understand for people with limited english, (or for translation tools to handle).
I similarly tried to avoid using `re-enable`, but I'm happy to follow recommendations.

Reply:
I've reworded this a bit. Is the new version clearer?

Reply:
Yep thanks